src/version.rs

```diff
@@ -1 +1 @@
-pub static LATEST: &str = "0.1";
+pub static LATEST: &str = "1";
```

Comment:
what makes us jump from version 0.1 to 1...?

Reply:
Thanks, it appears to have been an artefact of merge with main at some point, reverted.